index.bs

```diff
@@ -850,6 +867,27 @@ example of the latter, when the user is accessing the [RP] from a given client f
 use a [=roaming authenticator=] which was originally registered with the [RP] using a different client.
 
 
+### Credential Private Key Storage Option enumeration (enum <dfn enum>KeyStorage</dfn>) ### {#keyStorage}
+
+<pre class="idl">
+    enum KeyStorage {
+        "onAuthenticator",
```

Comment:
IMO this text is confusing and very likely to be misunderstood. Much better to have choices like say "resident" and "nonresident", and to use text similar to my suggestion on line 317. Otherwise a naive reader of this text might think it is okay to export a private key in the clear and store it on a server.